### PR TITLE
Fix Google and Office 365 help text formatting

### PR DESCRIPTION
**Pull request by jasonblais**
#### Summary
Fix Google and Office 365 help text formatting.

Removes "<>" tags in four locations

#### Ticket Link
N/A

#### Checklist
- [X] Includes text changes and localization files updated

### CODE REVIEW

**Comment by jwilander** [comment]
+1

File: webapp/components/admin_console/oauth_settings.jsx [hunk 1]
```diff
@@ -385,15 +385,15 @@ export default class OAuthSettings extends AdminSettings {
...
+                    defaultMessage="<ol><li><a href='https://login.microsoftonline.com/'>Log in</a> to your Microsoft or Office 365 account. Make sure it's the account on the same <a href='https://msdn.microsoft.com/en-us/library/azure/jj573650.aspx#Anchor_0'>tenant</a> that you would like users to log in with.</li><li>Go to <a href='https://apps.dev.microsoft.com'>https://apps.dev.microsoft.com</a>, click <strong>Add an app</strong> and use <strong>Mattermost - your-company-name</strong> as the application name.</li><li>Under <strong>Application Secrets</strong>, click <strong>Generate New Password</strong> and save it to later complete the field below.</li><li>Under <strong>Platforms</strong>, click <strong>Add Platform</strong>, choose <strong>Web</strong> and enter <strong>your-mattermost-url/signup/office365/complete</strong> (example: http://localhost:8065/signup/office365/complete) under <strong>Redirect URIs</strong>. Also uncheck <strong>Allow Implicit Flow</strong>.</li><li>Finally, click <strong>Save</strong> and complete the <strong>Application ID</strong> and <strong>Application Secret Password</strong> fields below.</li></ol>"
```
Comment (esethna) [inline webapp/components/admin_console/oauth_settings.jsx]: 1. Before clicking "Add an App" you need to click "go to app list". Propose `...click **Go to app list** > **Add an app**...`
2. Propose: `click Generate New Password and save it to later complete the field below` > `click Generate New Password and paste it below`

File: webapp/components/admin_console/oauth_settings.jsx [hunk 2]
```diff
@@ -385,15 +385,15 @@ export default class OAuthSettings extends AdminSettings {
...
...google.com</a>, click <strong>Credentials</strong> in the left hand sidebar and enter <strong>Mattermost - your-company-name</strong> as the project name.</li><li>Click the <strong>OAuth consent screen</strong> header and enter <strong>Mattermost</strong> as the <strong>Product name to show users</strong>. Click <strong>Save</strong>.</li><li>Under the <strong>Credentials</strong> header, click <strong>Create credentials</strong>, choose <strong>OAuth client ID</strong> and select <strong>Web Application</strong>.</li><li>Under <strong>Restrictions</strong> and <strong>Authorized redirect URIs</strong> enter <strong>your-mattermost-url/signup/google/complete</strong> (example: http://localhost:8065/signup/google/complete). Click <strong>Create</strong>.</li><li>Save the <strong>client ID</strong> and <strong>client secret</strong> to later complete the fields below.</li><li>Finally, go to <a href='https://console.developers.google.com/apis/api/plus/overview'>Google+ API</a> and click <strong>Enable</strong>. This might take a few minutes to propagate through Google's systems.</li><li>Complete the <strong>Client ID</strong> and <strong>Client Secret</strong> fields below.</li></ol>"
```
Comment (esethna) [inline webapp/components/admin_console/oauth_settings.jsx]: 1. Propose: `enter Mattermost - your-company-name as the project name.` > `enter "Mattermost - your-company-name" as the <strong>Project Name</strong>.`

**Comment by jasonblais** [comment]
Needs a rebase - closing to resubmit as it's simpler

**Pull request closed without merging**